spec/lib/auth0/api/v2/clients_spec.rb

```diff
@@ -5,18 +5,62 @@
     dummy_instance.extend(Auth0::Api::V2::Clients)
     @instance = dummy_instance
   end
+  
   context '.clients' do
     it { expect(@instance).to respond_to(:clients) }
     it { expect(@instance).to respond_to(:get_clients) }
+
     it 'is expected to send get request to /api/v2/clients/' do
-      expect(@instance).to receive(:get).with('/api/v2/clients', fields: nil, include_fields: nil)
+      expect(@instance).to receive(:get).with(
+        '/api/v2/clients',
+        fields: nil,
+        include_fields: nil,
+        page: nil,
+        per_page: nil
+      )
       expect { @instance.clients }.not_to raise_error
     end
+
     it 'is expected to send get request to /api/v2/clients?fields=name' do
-      expect(@instance).to receive(:get).with('/api/v2/clients', include_fields: true, fields: [:name])
-      expect { @instance.clients(fields: [:name], include_fields: true) }.not_to raise_error
+      expect(@instance).to receive(:get).with(
+        '/api/v2/clients',
+        include_fields: true,
+        fields: 'name',
+        page: nil,
+        per_page: nil
+      )
+      expect {
+        @instance.clients(fields: 'name', include_fields: true)
+      }.not_to raise_error
+    end
+
+    it 'is expected to convert fields param from Array to string' do
+      expect(@instance).to receive(:get).with(
+        '/api/v2/clients',
+        include_fields: true,
+        fields: 'name,app_type',
+        page: nil,
+        per_page: nil
+      )
+      expect {
+        @instance.clients(fields: ['name','app_type'], include_fields: true)
+      }.not_to raise_error
+    end
+
+    it 'is expected to add pagination' do
```

Comment:
This is asserting something about adding pagination, but only tests that an error wasn't raised. Is there an indicator that the params weren't silently ignored?

Reply:
So ... that first `expect` line sets up the second one. If you change the `page` value in `with()` to a different value, it fails.